src/components/PlaylistPage/MobileComponents.js

```diff
@@ -1,7 +1,7 @@
 import React, {PropTypes} from 'react';
 import {LessonFilterContainer} from '../Filter/LessonFilter';
 import LevelNavigation from './LevelNavigation';
-import MobileButtonsRow from './MobileButtonsRow';
+import {MobileButtonsRowContainer} from './MobileButtonsRow.js';
```

Comment:
Denne linja får bygget til å feile, fordi MobileButtonsRow.js gjør `export default` og her importeres det som named import. Så for å få bygget til å bygge uten feil, så må klammene vekk i denne linja.

Men når det er sagt, hvorfor ble denne linja endret? For MobileButtonsRow eksporterer jo ingen Container... og hvorfor ble `.js` lagt på?